### PR TITLE
Added reference to QualifiedTable to table args

### PR DESCRIPTION
**Pull request by unfernandito**
Added reference to QualifiedTable to table args in create_event_trigger 
TODO:
Make the reference to QualifiedTable Doc.

### Affected components

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System
- [ ] Tests
- [ ] Other (list it)

#### Catalog upgrade

Does this PR change Hasura Catalog version?
- [x] No
- [ ] Yes
  - [ ] Updated docs with SQL for downgrading the catalog 

#### Metadata

Does this PR add a new Metadata feature?
- [x] No
- [ ] Yes
  - Does `run_sql` auto manages the new metadata through schema diffing?
    - [ ] Yes
    - [ ] Not required
  - Does `run_sql` auto manages the definitions of metadata on renaming?
    - [ ] Yes
    - [ ] Not required
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?
    - [ ] Yes
    - [ ] Not required

#### GraphQL
- [x] No new GraphQL schema is generated
- [ ] New GraphQL schema is being generated:
   - [ ] New types and typenames are correlated
   
   

#### Breaking changes

- [x] No Breaking changes
- [ ] There are breaking changes:

  1. Metadata API

     Existing `query` types:
     - [ ] Modify `args` payload which is not backward compatible
     - [ ] Behavioural change of the API
     - [ ] Change in response `JSON` schema
     - [ ] Change in error code
     

  2. GraphQL API

     Schema Generation:
     
     
     - [ ] Change in any `NamedType`
     - [ ] Change in table field names
     

     Schema Resolve:-
     
     - [ ] Change in treatment of `null` value for any input fields 
     

  3. Logging

     - [ ] Log `JSON` schema has changed
     - [ ] Log `type` names have changed

### CODE REVIEW

File: docs/graphql/manual/api-reference/schema-metadata-api/event-triggers.rst [hunk 1]
```diff
@@ -74,8 +77,8 @@ Args syntax
      - Name of the event trigger
    * - table
      - true
-     - :ref:`TableName <TableName>`
-     - Name of the table
+     - [ :ref:`TableName <TableName>` | :ref:`QualifiedTable <QualifiedTable>` ]
```
Comment (tirumaraiselvan) [inline docs/graphql/manual/api-reference/schema-metadata-api/event-triggers.rst]: As seen here: https://github.com/hasura/graphql-engine/blob/b84db36ebb51acd5b51e1254c103f3097a7c2358/server/src-lib/Hasura/Events/Lib.hs#L97

The schema for table is actually always `QualifiedTable` in the payload. So what we had before here was actually incorrect.

Comment (unfernandito) [reply docs/graphql/manual/api-reference/schema-metadata-api/event-triggers.rst]: So, should i change the documentation again to set QualifiedTable only?

Comment (tirumaraiselvan) [reply docs/graphql/manual/api-reference/schema-metadata-api/event-triggers.rst]: Yes, pls. Thanks!

File: docs/graphql/manual/api-reference/schema-metadata-api/event-triggers.rst [hunk 2]
```diff
@@ -74,8 +77,8 @@ Args syntax
      - Name of the event trigger
    * - table
      - true
-     - :ref:`TableName <TableName>`
-     - Name of the table
+     - [ :ref:`QualifiedTable <QualifiedTable>` ]
```
Comment (unfernandito) [inline docs/graphql/manual/api-reference/schema-metadata-api/event-triggers.rst]: I set QualifiedTable as only type for table args

File: docs/graphql/manual/api-reference/schema-metadata-api/event-triggers.rst [hunk 3]
```diff
@@ -74,8 +77,8 @@ Args syntax
...
+     - Object with Table Name and Schema
```
Comment (marionschleifer) [inline docs/graphql/manual/api-reference/schema-metadata-api/event-triggers.rst]:
```suggestion
     - Object with table name and schema
```

**Pull request merged**